### PR TITLE
refine look of traffic lights

### LINKED ISSUE #354: [dashboard] Using colours in the progress meter
Grazina thought it was odd that the progress meter uses three colours (green, blue, and grey) rather than two.

This is especially since the first time you see the meter, you are already on step two. At that point there’s one step in each colour, making it less obvious that they’re progress colours (rather than, say, step colours).

- Also look at the amount of colour (more delicate)

---

Copied from Trello card: https://trello.com/c/MUQPy1BO/298-dashboard-using-colours-in-the-progress-meter
Labels: design

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* removed the blue colour
* made the dots smaller
* removed the 1,2,3 label … can be removed if we like this better
* other design tweaks

## QA

1. go to the initial repo enrollment and check if the traffic lights match the design in the related issue

## Issue/Card

Fixes #354 

## Screenshots

image

### CODE REVIEW

File: src/common/components/traffic-lights/index.js [hunk 1]
```diff
@@ -45,13 +45,13 @@ export default class TrafficLights extends Component {
 
     this.state = {
       signals: [{
-        label: '1',
+        label: '',
```
Comment (bartaz) [inline src/common/components/traffic-lights/index.js]: If we don't use the label, let's make it optional (in Signal component), so we don't have to specify it here at all.

Comment (earnubs) [inline src/common/components/traffic-lights/index.js]: Or give it default props.

File: src/common/components/traffic-lights/traffic-light.css [hunk 2]
```diff
@@ -8,15 +8,14 @@ $signal-colour-default: $mid-grey;
 
 .signal {
   background: $signal-colour-default;
-  border-radius: 50%;
+  border-radius: 16px;
```
Comment (earnubs) [inline src/common/components/traffic-lights/traffic-light.css]: 50% should still work here, or 100%, no?

Comment (bartaz) [inline src/common/components/traffic-lights/traffic-light.css]: I think it was just following spec: https://github.com/canonical-ols/build.snapcraft.io/issues/354#issuecomment-285355591

But yes, 50% would just work fine (as it's 50% of 32px)

File: src/common/components/traffic-lights/index.js (hunk 1 above)
Comment (pmahnke) [inline src/common/components/traffic-lights/index.js]: @earnubs  and @bartaz I updated this... did I do it ok

**Comment by jamiedawsonyoung** [comment]
[Using colours in the progress meter #354](https://trello.com/c/q3HbBkTA/601-using-colours-in-the-progress-meter-354)

**Comment by pmahnke** [comment]
@bartaz thanks... however, I don't have write access and cannot merge.

**Pull request merged**